app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -51,7 +54,11 @@ class SelectionInteractionViewModel private constructor(
     interaction.customizationArgsMap["maxAllowableSelectionCount"]?.signedInt
       ?: minAllowableSelectionCount
   }
-  private val selectedItems: MutableList<Int> = mutableListOf()
+
+  val selectedItems: MutableList<Int> = mutableListOf()
+  val selectedAnswer: MutableList<Int> =
```

Comment:
Here also I am storing selectedAnswers and showing it in UI, but it's not working. I have also added a seperate attribute `selectedItem` in `SelectionInteractionView` but still it's not working.